requirements.txt

```diff
@@ -16,3 +16,4 @@ xarray-datatree==0.0.6
 psutil>=5.9.1
 more-itertools==8.13.0
 geopy
+scikit-image
```

Comment:
This is a large library, so let's make it optional so that only people who want to use the functions that contain skimage functions will need to install them.